modules/ipc/src/zenoh/raw_subscriber.cpp

```diff
@@ -77,10 +77,10 @@ RawSubscriber::RawSubscriber(SessionPtr session, TopicConfig topic_config, DataC
   }
 
   auto sub_options = ::zenoh::ext::SessionExt::AdvancedSubscriberOptions::create_default();
-  if (session_->config.cache_size > 0) {
+  if (session_->cache_size > 0) {
```

Comment:
you found a bug from my previous refactor, thanks, this should be:

```suggestion
  if (config>cache_size.has_value() && *config->cache_size > 0) {
```